src/types/form.ts

```diff
@@ -139,10 +139,10 @@ export interface IFormSchema extends IForm, Document {
    * @param newOwner the new owner of the form. Similarly retrieved outside of method to force correct validation.
    * @returns updated form
    */
-  transferOwner(
+  transferOwner<T = IFormSchema>(
     currentOwner: IUserSchema,
     newOwner: IUserSchema,
-  ): Promise<IFormSchema>
+  ): Promise<T>
```

Comment:
I find it strange that we still need to reference `T`

Reply:
This is for the typing differences when called from a `IPopulatedForm` object vs a `IFormSchema` object